src/Microsoft.OData.Core/Microsoft.OData.Core.csproj

```diff
@@ -103,4 +103,8 @@
       <Version>6.0.3</Version>
     </PackageReference>
   </ItemGroup>
+

```

Comment:
How did this change come about?

Reply:
I think it comes after running the strings generator. Let me remove it.